types/lodash/tsconfig.json

```diff
@@ -6,6 +6,7 @@
         ],
         "noImplicitAny": true,
         "noImplicitThis": true,
+        "strictFunctionTypes": false,
```

Comment:
We support `strictFunctionTypes` now - there is no need to set it to `false`. In fact, it is set to `true` 2 lines down. However if you want to move that line up here (so they are in alphabetical order) I won't object.